webapp/shop/api/ua_contracts/advantage_mapper.py

```diff
@@ -34,6 +35,7 @@
     InvoiceSchema,
     EnsurePurchaseAccountSchema,
 )
+from webapp.shop.schemas import MagicAttachTokenSchema
```

Comment:
Are we using `MagicAttachTokenSchema` in this file? If not can be removed.

Reply:
Ok yes. Nice catch. This is leftover from the changes that happened to the API and UX flow. Needs to be removed